go.mod

```diff
@@ -211,4 +211,4 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 )
 
-replace antrea.io/ofnet v0.6.0 => github.com/wenyingd/ofnet v0.0.0-20220817031400-cb451467adc1
+replace antrea.io/ofnet => github.com/graysonwu/ofnet v0.0.0-20230307193942-e7cecbbf5fc7
```

Comment:
I suppose we need to merge the other PR first

Reply:
Yeah, now that PR has been merged and updated go.mod.